src/applications/facility-locator/components/ResultsList.jsx

```diff
@@ -1,14 +1,29 @@
 /* eslint-disable arrow-body-style */
 import React, { Component } from 'react';
+import _ from 'lodash';
 import PropTypes from 'prop-types';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import { updateSearchQuery, searchWithBounds } from '../actions';
 import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
 import SearchResult from './SearchResult';
 import Pagination from '@department-of-veterans-affairs/formation-react/Pagination';
+import { facilityTypes } from '../config';
 
 class ResultsList extends Component {
+  shouldComponentUpdate(nextProps) {
```

Comment:
not a fan of using this but I didn't want to refactor the redux state which is for the most part just spreading action payloads into new objects.